logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -256,7 +256,9 @@ internal class ConnectionDataSource(
                         accessRole = emptyList(),
                         receiptMode = ConversationEntity.ReceiptMode.DISABLED,
                         messageTimer = null,
-                        userMessageTimer = null
+                        userMessageTimer = null,
+                        otrArchived = false,
+                        otrArchivedRef = ""
```

Comment:
```suggestion
                        archived = false,
                        archivedRef = ""
```

I would suggest to drop the `otr` prefix because it's a legacy naming which just doesn't make sense. OTR is an abbreviation for "off the record" but that has nothing to do with we do now. OTR was used in a few places two differentiate between encrypted and non-encrypted features.

Reply:
Oh great! I was not aware of that!